stdlib/json.gr

```diff
@@ -101,8 +101,8 @@ provide enum IndentationFormat {
    *
    * ```json
    * {
-   *   "currency":·"€",
-   *   "price":·99.9
+   *   "currency": "€",
+   *   "price": 99.9
```

Comment:
I think this still needs to be tabs at the start, right? It seems like 2 spaces when I highlight in the github UI

Reply:
```suggestion
   * 	"currency": "€",
   * 	"price": 99.9
```